src/Domain/Entity/Service.php

```diff
@@ -153,4 +154,28 @@ public function isActiveAt(DateTimeImmutable $dateTime): bool
     {
         return (!$this->startDate || $this->startDate <= $dateTime) && (!$this->endDate || $dateTime < $this->endDate);
     }
+
+    /**
+     * Returns true if this service is part of a TV network
+     */
+    public function isTv(): bool
+    {
+        // Network can be null, but unfetched should throw an exception
+        if ($this->network && $this->getNetwork()->getMedium() === NetworkMediumEnum::TV) {
```

Comment:
Thoughts on these being single liners?

`return $this->network && $this->getNetwork()->getMedium() === NetworkMediumEnum::TV`

Counter-bikeshedding - should network have the isTV / isRadio functions, then service has the vanity shortcut `return $this->network && $this->getNetwork()->isTv()`?